src/ucp/core/ucp_request.h

```diff
@@ -36,6 +36,7 @@ enum {
     UCP_REQUEST_FLAG_RNDV                 = UCS_BIT(9),
     UCP_REQUEST_FLAG_OFFLOADED            = UCS_BIT(10),
     UCP_REQUEST_FLAG_BLOCK_OFFLOAD        = UCS_BIT(11),
+    UCP_REQUEST_FLAG_RNDV_MRAIL           = UCS_BIT(12),
```

Comment:
do we really need flag? can we use (rndv.mrail==NULL) instead?

Reply:
we need, at least for logging

Reply:
can't we check other fields in logging? like ep_config->num_rndv_lanes for example?